scripts/bert/staticbert/index.rst

```diff
@@ -0,0 +1,166 @@
+Example Usage of Exporting Hybridizable BERT
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For exporting hybridizable BERT, we mean the BERT with fixed input embedding size and sequence length can be exported through
+a static shape based implementation of hybridblock based BERT. By using this, we can export a block based BERT model.
+
+Please follow the steps below for exporting the model.
+
+Step 1: Load dataset. An example can be found in 'static_export_squad.py'.
+
+
+Step 2: create a hybridizable task guided model using BERT:
+
+.. code-block:: console
+
+    $ class StaticBertForQA(HybridBlock)
+
+An example can be found in 'static_bert_for_qa_model.py'.
+
+
+Step 3: specify the static input size and sequence length of the input data via environment variables
+in the training script:
+
+.. code-block:: console
+
+    $ os.environ['SEQLENGTH'] = str(args.seq_length)
+    $ os.environ['INPUTSIZE'] = str(args.input_size)
+
+An example can be found in 'static_export_squad.py'.
+
+
+Step 4: hybridize the model in the script:
+
+.. code-block:: console
+
+    $ net = StaticBertForQA(bert=bert)
+    $ net.hybridize(static_alloc=True, static_shape=True)
+
+An example can be found in 'static_export_squad.py'.
+
+
+Step 5: export trained model:
+
+.. code-block:: console
+
+    $ net.export(os.path.join(args.output_dir, 'static_net'), epoch=args.epochs)
+
+To export the model, in 'static_export_squad.py', set export=True.
+
```

Comment:
Can we add an example usage like the following?
```
python static_export_squad.py --export --model_parameters /PATH/TO/PARAMS/FROM/BERTQA/BLOCK 
```
and explain that this loads the the StaticBERTQA HybridBlock with parameter trained by a normal BERTQA Block, and export the HybridBlock to json format.